server/routes/middleware.js

```diff
@@ -59,4 +60,27 @@ const hasUserSignedBody = async (req, res, next) => {
   next()
 }
 
-module.exports = { partialReqCheck, reqCheck, checkExistence, hasUserSignedBody }
+const validateID = async (req, res, next) => {
+  const { id: unvalidatedID } = req.body
+  let u
+  if (unvalidatedID.substr(0, 2) === '0x') {
+    if (!ethers.utils.isAddress(unvalidatedID)) {
+      return res.status(StatusCodes.BAD_REQUEST).json({ error: `invalid Address: ${unvalididatedID}` })
+    } else {
+      // find by address
+      u = await User.findByAddress({ unvalidatedID })
```

Comment:
address is parameter name, not unvalidatedID

Reply:
updated 🙏